scheduler/scheduler.go

```diff
@@ -118,7 +118,7 @@ type Scheduler interface {
 	Runner
 
 	// Submit submits an app, creating it or updating it as necessary.
-	Submit(context.Context, *App) error
+	Submit(context.Context, *App, EventChan) error
```

Comment:
yea, i'd also like to include "levels" on the events so we have a way of publishing some of these (like the stabilization event) to the empire event stream.

i'll take a crack at this today